src/workspaces/list/NoContentMessage.test.tsx

```diff
@@ -7,37 +7,21 @@ import { NoContentMessage } from 'src/workspaces/list/NoContentMessage';
 import { WorkspaceFilterValues } from 'src/workspaces/list/WorkspaceFilters';
 
 describe('NoContentMessage', () => {
-  const defaultWorkspace: CategorizedWorkspaces = {
-    myWorkspaces: [defaultGoogleWorkspace],
-    newAndInteresting: [],
-    featured: [],
-    public: [],
-  };
-
-  const noWorkspaces: CategorizedWorkspaces = {
-    myWorkspaces: [],
-    newAndInteresting: [],
-    featured: [],
-    public: [],
-  };
-
-  const keywordFilters: WorkspaceFilterValues = {
-    keywordFilter: 'nothingMatchesThis',
-    accessLevels: [],
-    tab: 'myWorkspaces',
-    tags: [],
-  };
-
-  const featuredFilters: WorkspaceFilterValues = {
-    keywordFilter: '',
-    accessLevels: [],
-    tab: 'featured',
-    tags: [],
-  };
-
   it('displays loading message if workspaces are loading', () => {
     // Arrange
-    const props = { loadingWorkspaces: true, workspaces: defaultWorkspace, filters: keywordFilters };
+    const workspaces: CategorizedWorkspaces = {
+      myWorkspaces: [defaultGoogleWorkspace],
+      newAndInteresting: [],
+      featured: [],
+      public: [],
+    };
+    const filters: WorkspaceFilterValues = {
+      keywordFilter: 'nothingMatchesThis',
```

Comment:
Do you mind removing the keywordFilter to make it clear that it's not related to this test case?